### PR TITLE
Adds the easyrdf library into the dependencies and uses it

### PR DESCRIPTION
**Pull request by shinobu**
Adds the easyrdf library into the dependencies and uses it to parse rdfa in the RdfaWrapper (the rdfa import through arc2 did not work)
furthermore the RdfaWrapper got actualized and does now handle links
like the LinkedDataWrapper handled them.

### CODE REVIEW

**Comment by pfrischmuth** [comment]
- [ ] manual review commits
- [ ] check unit tests:
  - develop Branch:
  - Pull Request:
- [ ] check integration tests mysql:
  - develop Branch:
  - Pull Request:
- [ ] check integration tests virtuoso:
  - develop Branch:
  - Pull Request:

**Comment by shinobu** [comment]
Added a phpunit test for the RdfaWrapper

**Comment by pfrischmuth** [comment]
- [ ] manual review commits:
- [ ] check unit tests:
  - develop Branch:
  - Pull Request:
- [ ] check integration tests mysql:
  - develop Branch:
  - Pull Request:
- [ ] check integration tests virtuoso:
  - develop Branch:
  - Pull Request:
- [ ] further manual tests:
  - [ ] consider: were unit/integration tests created that test the feature/fix?

File: library/Erfurt/Wrapper/RdfaWrapper.php [hunk 1]
```diff
@@ -9,110 +9,282 @@
...
+        }
+
+        if ((substr($uri, 0, strlen($pattern)) === $pattern)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    private function _getHttpClient ($uri, $options = array())
```
Comment (pfrischmuth) [inline library/Erfurt/Wrapper/RdfaWrapper.php]: style: remove space

File: library/Erfurt/Wrapper/RdfaWrapper.php [hunk 2]
```diff
@@ -9,110 +9,282 @@
...
+                        $data[$s][$p] = array();
+                    }
+                    $data[$s][$p][] = $oSpec;
+                }
+            }
+        }
+        return  $data;
+    }
+
+    private function _matchUri ($pattern, $uri)
```
Comment (pfrischmuth) [inline library/Erfurt/Wrapper/RdfaWrapper.php]: style: remove space

File: library/Erfurt/Wrapper/RdfaWrapper.php [hunk 3]
```diff
@@ -9,110 +9,282 @@
...
+                    true
+                );
+
+                $response = $client->request();
+
+                return $this->_handleResponse($client, $response);
+            default:
+                return false;
+        }
+    }
```
Comment (pfrischmuth) [inline library/Erfurt/Wrapper/RdfaWrapper.php]: style: add new line between methods

**Comment by pfrischmuth** [comment]
OK, but some style issues still need to be resolved prior merging.

**Comment by shinobu** [comment]
isn't the second style issue actually correct? it's a newline prior to a new function

**Comment by pfrischmuth** [comment]
I was relating to the space after the method name.

**Comment by shinobu** [comment]
done

if those 2 style-rules are important we should add them to our codesniffer tests

**Comment by pfrischmuth** [comment]
No, the spaces after the method names are still there.

**Comment by shinobu** [comment]
it should be done now

**Comment by pfrischmuth** [comment]
- [x] manual review commits: OK

**Comment by pfrischmuth** [comment]
OK

**Pull request merged**